ubg/src/mafia/MafiaDay.js

```diff
@@ -225,6 +237,7 @@ MafiaDay.propTypes = {
 
 const mapStateToProps = (state) => ({
   userUid: state.currentUser.uid,
+  userAlive: state.currentUser.alive,
```

Comment:
Current user is the information that firebase returns of the user, I don't think it has alive, unless you added it in another PR.